kube/src/api/metadata.rs

```diff
@@ -13,7 +13,7 @@ use serde::{Deserialize, Serialize};
 /// - .metadata.resource_version
 ///
 /// This avoids a bunch of the unnecessary unwrap mechanics for apps
-pub trait Meta: Metadata {
```

Comment:
:+1:

Reply:
Tried porting this over to the linked replacement pr, but it is currently depended on by the controller.